src/tutorials/0007-verifying-storage-on-filecoin/02.md

```diff
@@ -0,0 +1,25 @@
+---
+    title: "Preparing and transferring data"
+    type: "multiple-choice" # can be "text", "code", "file-upload" or "multiple-choice"
+---
+
+## Preparing data for storage
+
+Before a system file (e.g. `puppy.gif`) can be stored on the Filecoin network, it must first be transformed into a Filecoin Piece.
+
+In the first stage of this transformation, the system file is chunked up with UnixFS to create an IPLD DAG (Directed Acyclic Graph). You can learn more about DAGs (a form of merkle tree) in our [Decentralized Data Structures](https://proto.school/#/data-structures) tutorial. This IPLD DAG has a _payload CID_, identical to an IPFS CID, which represents the root of the DAG.
+
+The IPLD DAG is then serialized to a [CAR file](https://github.com/ipld/specs/blob/master/block-layer/content-addressable-archives.md#summary) and bitpadded to make a Filecoin Piece. This piece has a unique _piece CID_, also known as a CommP (Piece Commitment).
```

Comment:
I am not certain that `piece CID` and `CommP` are the exact same thing, better check with someone else as well.

Reply:
Got this info from Laser. Thanks!